### ISSUE TITLE
Connections still being logged (kinda?)

### ISSUE THREAD

**Issue by A5ho9999** [adryd325/sneaky#5]
Seems the connections are still being logged in a way, this warning pops up instead of the normal disconnected message however. (the connecting account is 'cuute').

Config Settings;
```
dont-log-unauthed-client-disconnects=true
dont-log-unauthed-server-disconnects=false
hide-player-list=true
hide-server-ping-data=true
new-connection-rate-limit=5
rate-limit-new-connections=true
```

```
[12:43:43] [Netty Epoll Server IO #2/WARN]: Failed to initialize a channel. Closing: [id: 0x234a71da, L:/*:25565 - R:/3.64.214.17:55898]
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.get(Object)" is null
        at com.adryd.sneaky.Sneaky.checkAllowConnection(Sneaky.java:56) ~[sneaky-mc1.20.1-1.0.3.jar:?]
        at net.minecraft.class_3242$1.handler$eoh000$sneakyserver$beforeAssignNetworkHandler(class_3242.java:1014) ~[server-intermediary.jar:?]
        at net.minecraft.class_3242$1.initChannel(class_3242.java) ~[server-intermediary.jar:?]
        at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:938) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:609) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1463) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1115) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:650) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:514) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:403) ~[netty-transport-classes-epoll-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]

**Comment by adryd325** [comment]
ah,, thank you this is my fault, i'll fix this when i'm home asap

**Comment by adryd325** [comment]
disable the rate limit new connections option for now

**Comment by adryd325** [comment]
I'm on a trip and dont have access to a computer, could you test the release I just made on modrinth and see if this is fixed

**Comment by A5ho9999** [comment]
Thanks, I won't be able to check for a bit though as waiting on mod conflict to be handled before can run the server again. I appreciate the dedication to do that from mobile though.

**Comment by A5ho9999** [comment]
Seems to be fixed, server has been up for about 12 hours or so and haven't had one popup. Will close this tomorrow sometime just in-case it does popup in that time

**Comment by A5ho9999** [comment]
Never mind, it appeared again. However this one appeared when I connected to the server.

```
[02:05:00] [Netty Epoll Server IO #3/WARN]: Failed to initialize a channel. Closing: [id: 0xa30380d8, L:/*:25565 - R:/*:37966]
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.get(Object)" is null
        at com.adryd.sneaky.Sneaky.checkAllowConnection(Sneaky.java:56) ~[sneaky-mc1.20.1-1.0.4+1e9488a0.jar:?]
        at net.minecraft.class_3242$1.handler$ffa000$sneakyserver$beforeAssignNetworkHandler(class_3242.java:1014) ~[server-intermediary.jar:?]
        at net.minecraft.class_3242$1.initChannel(class_3242.java) ~[server-intermediary.jar:?]
        at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:938) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:609) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1463) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1115) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:650) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:514) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486) ~[netty-transport-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:406) ~[netty-transport-classes-epoll-4.1.82.Final.jar:?]
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.82.Final.jar:?]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.82.Final.jar:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]

**Comment by adryd325** [comment]
I fixed it properly in 1.0.6

**Comment by A5ho9999** [comment]
Yup, thanks!
